virt-v2v/pkg/server/server.go

```diff
@@ -21,6 +22,7 @@ var (
 // Operating System for the VM creation.
 func Start() error {
 	http.HandleFunc("/ovf", ovfHandler)
```

Comment:
do we still need the ovf endpoint ?

Reply:
In this PR yes, as we are still exposing the firmware to the ova, but that has also its own issues so this will be replaced in followup PR similar to the Bellas changes https://github.com/kubev2v/forklift/pull/1020